src/main/java/org/testng/SuiteRunnerWorker.java

```diff
@@ -65,16 +68,19 @@ private void runSuite(SuiteRunnerMap suiteRunnerMap /* OUT */, XmlSuite xmlSuite
       SuiteResultCounts counts = new SuiteResultCounts();
       counts.calculateResultCounts(xmlSuite, suiteRunnerMap);
 
-      StringBuilder bufLog = new StringBuilder("\n===============================================\n")
-          .append(xmlSuite.getName());
-      bufLog.append("\nTotal tests run: ")
-          .append(counts.m_total).append(", Failures: ").append(counts.m_failed)
-          .append(", Skips: ").append(counts.m_skipped);
+      StringBuilder bufLog = new StringBuilder(LINE).append(xmlSuite.getName());
```

Comment:
You can remove the string builder here: there is no loop and the string concatenation will do the same things.